.circleci/config.yml

```diff
@@ -145,18 +148,6 @@ commands:
       - run: make e2e
       - run: make e2e-filter-test
 
-  run_e2e_tests_nightly:
```

Comment:
this isn't needed anymore after adding the channel var.

Reply:
update.sh is defining `stable` / `nightly` in these two jobs, isn't that still needed?

Reply:
oops. you're right. we can remove the -nightly suffix, right?